src/zope/dublincore/tests/test_dcsv.py

```diff
@@ -13,10 +13,8 @@
 ##############################################################################
 """Test the Dublib Core Structured Value support functions.
 """
-import unittest
-
 from doctest import DocTestSuite
-from zope.dublincore.dcsv import encode, decode
+from zope.dublincore.dcsv import encode, decode  # noqa: F401
```

Comment:
I had to look up what F401 means. 

This says something's unused, and this is a test module, so it can't be here for exporting a public API, and flake8 supports `doctests = yes` so it can't be for the benefit of doctests, so why not remove the import?

Reply:
The import is needed, the noqa comment is not.